src/coreclr/jit/compiler.cpp

```diff
@@ -1262,8 +1262,12 @@ void DisplayNowayAssertMap()
             fout = _wfopen(strJitMeasureNowayAssertFile, W("a"));
             if (fout == nullptr)
             {
+#if !defined(HOST_WINDOWS)
+                // TODO: how do we print a `const char16_t*` portably?
```

Comment:
There are a lot more `%ws` in the jit codebase, why this one specifically is removed?

Reply:
e.g. 
```c
JITDUMP("Trying to unroll MemoryExtensions.Equals|SequenceEqual|StartsWith(op1, \"%ws\")...\n", str)
```

Reply:
This is the only one the compiler complained about when building [NativeAOT-LLVM](https://github.com/dotnet/runtimelab/tree/feature/NativeAOT-LLVM) on non-Windows systems.  I haven't dug into why that is, but my guess is that either other cases of `%ws` aren't included in the `NativeAOT-LLVM` build, or else they're being used with expressions which are of type `wchar_t*` rather than `char16_t*`; the former is portable when used with `%ws`, but the latter is not.

Reply:
The reason this doesn't warn in other cases is because they are are `JITDUMP`s, which compiles down to `logf`, which the compiler doesn't recognize as a `printf`-style function. I wonder if these other usages actually work properly on not-Windows.